client/domain/FeedbackParagraphObjectFactory.js

```diff
@@ -109,6 +109,16 @@ tie.factory('FeedbackParagraphObjectFactory', [
       return this._content;
     };
 
+    /**
+     * A getter for the line number where the error occurs.
+     *
+     * @returns {number}
+     */
+    FeedbackParagraph.prototype.getErrorLineNumber = function() {
+      var errorContentArray = this.getContent().split(' ');
```

Comment:
Should check that the paragraph is of the correct type first, and if not, raise error

Also, add dedicated tests for this method.